deepfence_frontend/apps/dashboard/src/api/generated/apis/CloudResourcesApi.ts

```diff
@@ -86,7 +86,7 @@ export class CloudResourcesApi extends runtime.BaseAPI implements CloudResources
             method: 'POST',
             headers: headerParameters,
             query: queryParameters,
-            body: requestParameters.ingestersCloudResource?.map(IngestersCloudResourceToJSON),
+            body: requestParameters.ingestersCloudResource.map(IngestersCloudResourceToJSON),
```

Comment:
need to fix this, this is a but with the code generator at the moment. we manually fix this type of occurrences in generated code. this code actually should be
`body: requestParameters.ingestersCloudResource?.map(IngestersCloudResourceToJSON),`